magda-gateway/package.json

```diff
@@ -73,6 +73,7 @@
     "@types/passport-facebook": "^2.1.8",
     "@types/passport-local": "^1.0.31",
     "@types/pg": "^6.1.41",
+    "@types/q": "^1.5.2",
```

Comment:
What does this do?

Reply:
`react-scripts` in our mono-repo actually used some old dependencies that other module uses due to hoisting.
The error was muted so it doesn't create trouble for 
this makes optional-chaining babel plugin not working as it requires higher version babel.
I set yarn workspace to stop hosting `react-scripts` and its dependencies so that `react-scripts` has access to the version it really needs.
However, some of `react-scripts`'s dependencies might not be available at top level `node_modules` anymore (see thi s [log](https://gitlab.com/magda-data/magda/-/jobs/475883471))--- this, consequently broke the CI pipeline. Thus, add the dependencies to impact modules' package.json to fix it: